web/src/user/settings/accessTokens/UserSettingsCreateAccessTokenPage.tsx

```diff
@@ -193,7 +193,7 @@ export class UserSettingsCreateAccessTokenPage extends React.PureComponent<Props
                     </button>
                     <Link
                         className="btn btn-secondary ml-1 e2e-create-access-token-cancel"
-                        to={`${this.props.match.url.replace(/\/new$/, '')}}`}
+                        to={`${this.props.match.url.replace(/\/new$/, '')}`}
```

Comment:
Thanks! Looking at it, I think it can be even simpler: 

```suggestion
                        to={this.props.match.url.replace(/\/new$/, '')}
```